catalog/ui/search-ui-app/pom.xml

```diff
@@ -17,8 +17,8 @@
     xmlns="http://maven.apache.org/POM/4.0.0"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <parent>
-        <groupId>ddf.ui.search</groupId>
-        <artifactId>search-ui</artifactId>
+        <groupId>ddf.ui</groupId>
+        <artifactId>ui</artifactId>
```

Comment:
Did we want to change this from search-ui to just ui? I think search-ui is still valid since this app has all the different search ui's in it